Objective-C/Internal/CBLValueExpression.m

```diff
@@ -82,6 +82,7 @@ static id dictionaryAsJSON(NSDictionary* dict) {
 
 static id arrayAsJSON(NSArray* array) {
     NSMutableArray *json = [NSMutableArray arrayWithCapacity: array.count];
```

Comment:
better to allocate the extra array operation space too. 

```suggestion
    NSMutableArray *json = [NSMutableArray arrayWithCapacity: array.count + 1];
```

Reply:
Thanks for catching that.